src/components/AnchorForCommentsOnly/types.ts

```diff
@@ -18,6 +18,9 @@ type AnchorForCommentsOnlyProps = ChildrenProps & {
 
     /** Press handler for the link, when not passed, default href is used to create a link like behaviour */
     onPress?: () => void;
+
+    /** Indicates whether an image is wrapped in an anchor (`<a>`) tag with an `href` link */
+    isLinkHasImage?: boolean;
```

Comment:
Can we change the wording of this to be `linkHasImage`?

Reply:
it's done